public/pages/CreateMonitor/components/LocalUriInput/LocalUriInput.js

```diff
@@ -0,0 +1,60 @@
+/*
+ *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Please update the copyright year for the new files 😉.

Reply:
Ah, thank you for catching this! Updated the year to 2021 in all files I modified.

Reply:
Thanks for updating the year. 👍